src/gui/src/main.cpp

```diff
@@ -112,6 +113,17 @@ int main(int argc, char* argv[])
 	}
 #endif
 
+    // Verify if build was built from a Git tag, or from a Git branch/untagged commit.
+    // TODO: Add CMake support.
+    // For now, this `QMessageBox` is disabled.
+#if defined(INPUTLEAP_GIT_UNTAG_WARNING)
+    if (1)
```

Comment:
Maybe it makes sense to add a yellow bar at the top of the GUI window and write message there? QMessageBox requires user interaction and will be very inconvenient.